src/venue.md

```diff
@@ -30,5 +30,13 @@ title: "Venue"
 
   <p>The venue is accessible both by BTS (Victory Monument) and the Airport Link (Phaya Thai).</p>
 
-  <a href="https://goo.gl/maps/vnLpSxt1zzw" target="_blank" class="card-venue__btn btn btn--primary">View in Google Maps</a>
+  <a href="https://goo.gl/maps/vnLpSxt1zzw" target="_blank" class="btn btn--primary">View in Google Maps</a>
+</section>
+
+<section class="venue-booking">
+  <h2>Staying There</h2>
+
+  <p>If you are interested in staying in the hotel during the event, you can use a dedicated booking portal provided directly by the hotel. The service is available to all attendees.</p>
```

Comment:
Can we mention that the portal provides a discounted rate for attendees?

Reply:
I am not sure how true this is 😅 I did not want to oversell it as likely Agoda might have a cheaper rate. 

The idea of booking through this link is the comfort of booking directly the hotel and the range of dates are pre-selected:

![image](https://user-images.githubusercontent.com/696529/193746379-6520c7b6-f92b-4521-8df0-b6589741a5db.png)

It might suite some people who do not care too much about finding the cheapest rate.

Reply:
it does seem *fairly* discounted. it's cheaper than agoda and their official site. i found a cheaper rate without breakfast on trip.com

Reply:
Fair enough. Then I added it in b54be0e